src/wymeditor/editor/undo-redo.js

```diff
@@ -23,6 +25,23 @@ WYMeditor.UndoRedo = function (wym) {
     // https://github.com/mightyiam/object-history
     undoRedo.history = new WYMeditor.EXTERNAL_MODULES
         .ObjectHistory(wym.getCurrentState());
+
+    wym.keyboard.combokeys.bind(
+        "mod+z",
+        function () {
+            wym.undoRedo.undo();
+            // Prevents native action. Caution: not covered by tests.
+            return false;
+        }
+    );
+    wym.keyboard.combokeys.bind(
+        jQuery.browser.mac ? "shift+meta+z" : "ctrl+y",
```

Comment:
I'm interested to get this tested on a mac. Do you think that this is correct, @winhamwr ?